Doc/library/os.rst

```diff
@@ -3997,7 +3997,7 @@ Linux extended attributes
 
 .. versionadded:: 3.3
 
-These functions are all available on Linux only.
+These functions are all available on Linux and Cygwin only.
```

Comment:
Documentation should say these are on Cygwin only in 3.13+.
```suggestion
.. versionchanged:: 3.13
   These functions are now available on Cygwin.

These functions are all available on Linux and Cygwin only.
```

Reply:
@encukou suggested below to simply leave out any docs changes, would that be okay for you too?